x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/index.tsx

```diff
@@ -164,14 +181,14 @@ export const HostList = () => {
       },
       {
         field: 'metadata.host.os.name',
-        name: i18n.translate('xpack.securitySolution.endpoint.host.list.os', {
+        name: i18n.translate('xpack.siem.endpointList.os', {
           defaultMessage: 'Operating System',
         }),
         truncateText: true,
       },
       {
         field: 'metadata.host.ip',
-        name: i18n.translate('xpack.securitySolution.endpoint.host.list.ip', {
+        name: i18n.translate('xpack.siem.endpointList.ip', {
```

Comment:
shouldn't all of these be the `securitySolution` naming?  This in particular looks like it was changed from `xpack.securitySolution.endpoint.host.list.ip`

Reply:
yeah - fixed now :)